README.md

```diff
@@ -33,15 +33,5 @@
 
 ## How to sync design tokens
 
-If you want to sync design tokens with Figma files you can use [Figma Tokens plugin](https://www.figma.com/community/plugin/843461159747178978/Figma-Tokens) and setup the plugin with the following information.
-
-1. Open Figma Tokens Plugin, go to `Settings` and select `Github` in Token Storage
-2. Add new credentials
-
-- **Name:** The name of the brand
-- **Personal Access Token:** you have to generate a token from Github. [Read how to do it](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic)
-- **Repository:** `Telefonica/mistica-design`
-- **Default Branch:** `production`
-- **File Path:** `tokens/brandName.json` (see files [here](./tokens/))
-
-![image](https://user-images.githubusercontent.com/6722153/166447592-e3d1b545-199d-4155-9024-2fb88351b444.png) 3. Finally, go to `Tokens`, select `Global` and `Apply to document` and clic in `Update`
+> [!NOTE]  
+> We're no longer using [Figma Tokens plugin](https://www.figma.com/community/plugin/843461159747178978/Figma-Tokens) to sync tokens with files. We use Figma API to update variables in our libraries. More about this proccess in [Figma tokens script README]().
```

Comment:
The url at the end is empty. Is this expected?

Reply:
Fixed